apps/meteor/client/views/omnichannel/directory/calls/CallTable.tsx

```diff
@@ -54,6 +55,21 @@ const CallTable: FC = () => {
 	const query = useQuery(debouncedParams, debouncedSort, userIdLoggedIn);
 	const directoryRoute = useRoute('omnichannel-directory');
 
+	const resolveDirectionLabel = useCallback(
+		(direction: Partial<IVoipRoom['direction']>) => {
```

Comment:
Partial is not necessary on this case, right? There is a specific portion of IVoipRoom defined as a type here (direction)... Partial is used when you don't know from what portion of the type must be used. I mean... It looks like an 'any' inside of a specific type defined!

Reply:
Yeah, it makes sense, but I don't want to keep it any since it can either be either (`inbound` | `outbound` | null). So I feel `Nullable` makes a bit more sense here. WDYT?

Reply:
Why direction can be a null value? 🤔

Reply:
Nullable type definition doesn't exist on TS... 🤔

Reply:
I was trying to add a fallback kind of thing here, hence I'd earlier added this extra typing to make the prop nullable. However, I now realise that its not required so I've removed it.

Reply:
Remember that (maybe not for this case :) but for others) you can create Utility types for your use cases, for example:
`Nullable<T> = T | null` and then use `Nullable<IVoipRoom['direction']>`